packer/setup_root.sh

```diff
@@ -19,9 +19,9 @@ INSTALL_K3S_SKIP_START="true"
 
 INSTALL_K3S_EXEC=""
 INSTALL_K3S_EXEC="$INSTALL_K3S_EXEC --disable=traefik"
-INSTALL_K3S_EXEC="$INSTALL_K3S_EXEC --kube-reserved cpu=500m,memory=1Gi,ephemeral-storage=1Gi"
-INSTALL_K3S_EXEC="$INSTALL_K3S_EXEC --system-reserved cpu=500m,memory=1Gi,ephemeral-storage=1Gi"
-INSTALL_K3S_EXEC="$INSTALL_K3S_EXEC --eviction-hard memory.available<0.5Gi,nodefs.available<10%"
+INSTALL_K3S_EXEC="$INSTALL_K3S_EXEC --kubelet-arg kube-reserved=cpu=500m,memory=1Gi,ephemeral-storage=1Gi"
+INSTALL_K3S_EXEC="$INSTALL_K3S_EXEC --kubelet-arg system-reserved=cpu=500m,memory=1Gi,ephemeral-storage=1Gi"
+INSTALL_K3S_EXEC="$INSTALL_K3S_EXEC --kubelet-arg eviction-hard=memory.available<0.5Gi,nodefs.available<10%"
```

Comment:
Have you verified that `--kubelet-arg` can be passed multiple times?

Otherwise LGTM

Reply:
Sharing the screenshots from our chat :-) 
![image](https://user-images.githubusercontent.com/5058132/226595217-1a90b542-b0e0-4021-a50f-7bdc95956620.png)
and Capacity is bigger than allocable
![image](https://user-images.githubusercontent.com/5058132/226595275-2e2da43b-d9d0-4a28-acd1-ea6592354862.png)

🎉